dashboard/test/ui/step_definitions/publicKeyCryptography_steps.rb

```diff
@@ -33,9 +33,8 @@
   @browser.find_element(:css, '.go-button').click
 end
 
-When /^I (?:hide|show) Eve's panel$/ do
-  @browser.find_element(:xpath, "//div[text()='Eve']").click
-  sleep 0.5
+When /^I open view "(Alice|Eve|Bob|All)"$/ do |view|
+  @browser.find_element(:xpath, "//button[text()[contains(.,'#{view}')]]").click
```

Comment:
It seems like we've had problems with xpath selectors lately. should we be using jquery instead?